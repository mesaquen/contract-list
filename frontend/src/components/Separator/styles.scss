.separator {
  background: var(--dividers-bg-bg-menu);
  height: 1px;
  width: 100%;
  margin: 0.25rem 0;
}
