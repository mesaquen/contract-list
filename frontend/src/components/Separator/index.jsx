import React from 'react'
import './styles.scss'

const Separator = () => {
  return <div className="separator"></div>
}

export default Separator
