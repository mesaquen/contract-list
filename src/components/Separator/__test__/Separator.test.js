import { cleanup, render } from '@testing-library/react'
import Separator from '../index'

beforeEach(cleanup)

describe('Separator', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Separator />)
    expect(asFragment()).toMatchSnapshot()
  })
})
