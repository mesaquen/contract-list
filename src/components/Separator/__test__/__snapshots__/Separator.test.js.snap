// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator renders correctly 1`] = `
<DocumentFragment>
  <div
    class="separator"
  />
</DocumentFragment>
`;
